McBopomofoTests/PreferencesTests.swift

```diff
@@ -144,9 +144,9 @@ class PreferencesTests: XCTestCase {
     }
 
     func testChineneConversionEngine() {
```

Comment:
func test**Chinese**ConversionEngine() {

Consider batch replace "hinene" to "hinese"?